index.css

```diff
@@ -0,0 +1,118 @@
+body, html {
```

Comment:
Overall not bad. There are few feedbacks that I noticed: 
1. No bird in the middle. 
2. Good use of icons for blue side contents 
3. No favicon (little website icon on browser tab)
4. Location of some elements and their sizes. 
5. Page does not respond well to re-sizing of browser window. 
Some of things are minor while others could be done better to make your page as similar as Twitter. I like the use of grid to position elements on the page, and you can also set grid within each div to place element more precisely. Good job and looking forward to see your next project!